tasks.py

```diff
@@ -268,6 +262,56 @@ def build(_c, debug=False):
         image.tag(constants.IMAGE, tag=latest_tag)
 
 
+@task
+def sbom(_c, debug=False):
```

Comment:
This is a side effect of `git diff`; I didn't actually change this section substantially, but I added another, similar function and made tweaks to this one but `git` cannot differentiate that.

Reply:
Git nuance I wouldn't have known otherwise.